nestkernel/simulation_manager.cpp

```diff
@@ -410,11 +466,19 @@ nest::SimulationManager::simulate( Time const& t )
     LOG( M_ERROR, "SimulationManager::simulate", msg );
     throw KernelException();
   }
+}
+
+void
+nest::SimulationManager::run( Time const& t )
+{
+  check_run( t );
 
   to_do_ += t.get_steps();
   to_do_total_ = to_do_;
+  assert( to_do_ != 0 );
 
-  const size_t num_active_nodes = prepare_simulation_();
+  // Check whether waveform relaxation is used on any MPI process
+  kernel().node_manager.check_wfr_use();
```

Comment:
I think it needs to happen only once after `update_delay_extrema_()`, but it should not hurt here either. The gap junction team must know.
@suku248 : ???